api/src/test/java/org/openmrs/CohortTest.java

```diff
@@ -9,10 +9,9 @@
  */
 package org.openmrs;
 
-import org.apache.commons.lang.ArrayUtils;
-import org.apache.commons.lang3.StringUtils;
-import org.junit.Assert;
-import org.junit.Test;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
```

Comment:
Are these changes required by the ticket?

Reply:
@dkayiwa these methods of Assert class where imported to be used in writing unit tests to avoid compilation errors while building

Reply:
please remove these changes from the PR